custom-recipes/send-mails-from-contacts-dataset/recipe.py

```diff
@@ -200,7 +220,7 @@ def confirm_string_column(column_name, schema):
                 if writer:
                     writer.write_row_dict(contact_dict)
             except Exception as e:
-                logging.exception("Send failed")
+                logging.exception("Send failed: %s", str(e))
```

Comment:
Explaining patch changes: may as well log the error as well as return it, makes debuging easier